cms/blocks.py

```diff
@@ -0,0 +1,16 @@
+"""Page blocks"""
+
+from wagtail.core import blocks
+from wagtail.images.blocks import ImageChooserBlock
+
+
+class CourseTeamBlock(blocks.StructBlock):
+    """
+    Block class that defines a course team member
+    """
+
+    name = blocks.CharBlock(max_length=100, help_text="Name of the course team member.")
+    bio = blocks.CharBlock(max_length=255, help_text="Short bio of course team member.")
```

Comment:
Are we okay with the 255 character limit on this?

Reply:
@pdpinch could you please answer this question.

Reply:
Yes, definitely. The size of the input box in the CMS is too small.

Reply:
Also, 255 characters is not enough for the existing sample copy at https://drive.google.com/file/d/1hYhbeNmzSVS-Hd3zBNzhcsRd2uefIlcJ/view 

The longest bio there looks to me to be about 700 characters. I'm not sure if this breaks the layout.